File: Mastery-Art-Front-End/resources/static/JS/components/Artist.js
import Art from './Art'

export default function Artist(artist) {
    return `
    <div class="artist__container">
        <h3 class="artist__artistName title">Artist: ${artist.artistName}</h3>
        <img src="${artist.artistImage}" class="artist__image">
    </div>


    <ul class="art">
            <h3>Art</h3>
            <li class="art">${Art(artist.art)}</li>
        </ul>

        `
}

File: Mastery-Art-API/src/main/java/com/wcci/mastery3/MasteryArt/models/Artist.java
package com.wcci.mastery3.MasteryArt.models;

import java.util.ArrayList;
import java.util.Collection;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Lob;
import javax.persistence.ManyToMany;

@Entity
public class Artist {
	@Id	
	@GeneratedValue
	private Long id;
	private String artistName;
	@Lob
	private String artistImage;
	@ManyToMany
	private Collection<Art> art;
	
	public Long getId() {
		return id;
	}
	public String getArtistName() {
		return artistName;
	}
	public String getArtistImage() {
		return artistImage;
	}
	
	public Collection<Art> getArt() {
		return art;
	}
	
	public void addGenreToGenres(Art artToAdd) {
		art.add(artToAdd);
	}	
	
	public Artist() {}
	
	public Artist(String artistName, String artistImage) {
		this.artistName = artistName;
		this.artistImage = artistImage;
		this.art = new ArrayList<Art>();
	}
	
	
}
